vmdb/app/models/container_image_registry.rb

```diff
@@ -0,0 +1,7 @@
+class ContainerImageRegistry < ActiveRecord::Base
+  include ReportableMixin
+
+  belongs_to  :ext_management_system, :foreign_key => "ems_id"
+  has_many :container_images, :dependent => :destroy # What about deleted registry but containers are still running
+  has_many :containers, :through => :container_images
```

Comment:
@abonas I admit I just wanted to see if this works but the GUI is set up for it as well.